apps/scoutgame/components/common/DraftDeveloperInfoModal/components/DraftDeveloperBidForm.tsx

```diff
@@ -322,20 +316,7 @@ function DraftDeveloperBidFormComponent({
           <Button onClick={onCancel} variant='outlined' color='secondary' size='large' disabled={isLoading}>
             Cancel
           </Button>
-          <Button
-            loading={isLoading}
-            onClick={handleSubmit}
-            variant='contained'
-            color='secondary'
-            size='large'
-            disabled={
-              !!customError ||
-              (selectedPaymentOption.currency !== 'DEV' &&
-                (!decentTransactionInfo || !('tx' in decentTransactionInfo) || 'error' in decentTransactionInfo)) ||
-              !!draftError ||
-              !!addressError
-            }
-          >
+          <Button loading={isLoading} onClick={handleSubmit} variant='contained' color='secondary' size='large'>
```

Comment:
did you remove the disabled prop since we won't be using this component anymore until next season?

Reply:
i think it was just an accident